tracer/src/Datadog.Trace/ContinuousProfiler/ContextTracker.cs

```diff
@@ -57,6 +59,14 @@ public void Set(ulong localRootSpanId, ulong spanId)
             WriteToNative(new SpanContext(localRootSpanId, spanId));
         }
 
+        public void SetEndpoint(ulong localRootSpanId, string endpoint)
+        {
+            if (IsEnabled && _isEndpointProfilingEnabled && !string.IsNullOrEmpty(endpoint))
```

Comment:
`IsEnabled` is false if `_isCodeHotspotsEnabled`. Does endpoint profiling require code hotspots?

Reply:
Correct